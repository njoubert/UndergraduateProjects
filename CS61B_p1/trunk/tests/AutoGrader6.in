Kaushik is not the student.
Paul is not the ta.
Nick is not the professor.
Kaushik lives in the fifth house.
Paul lives in the seventh house.
The professor lives in the seventh house.
The ta lives in the fifth house.
There is a studenthousing house.
Where does Nick live?
Who is the professor?
Who is the student?
What do you know about Kaushik?

