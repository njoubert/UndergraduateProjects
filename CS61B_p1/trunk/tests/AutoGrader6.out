1. Kaushik is not the student.
2. Paul is not the ta.
3. Nick is not the professor.
4. Kaushik lives in the fifth house.
5. Paul lives in the seventh house.
6. The professor lives in the seventh house.
7. The ta lives in the fifth house.
8. There is a studenthousing house.
Q: Where does Nick live?
A: Nick lives in the studenthousing house.
Q: Who is the professor?
A: Paul is the professor.
Q: Who is the student?
A: Nick is the student.
Q: What do you know about Kaushik?
A: Kaushik is the ta and lives in the fifth house.
